activestorage/test/controllers/representations/proxy_controller_test.rb

```diff
@@ -27,7 +28,7 @@ class ActiveStorage::Representations::ProxyControllerWithVariantsTest < ActionDi
     get rails_blob_representation_proxy_url(
       filename: @blob.filename,
       signed_blob_id: @blob.signed_id,
-      variation_key: ActiveStorage::Variation.encode(resize_to_limit: [100, 100]))
+      variation_key: ActiveStorage::Variation.encode(@transformations))
```

Comment:
Abstracting `@transformations` seems reasonable, but why update only this line (and not line 36, or the `"showing variant with invalid signed blob ID"` test, or other tests)?

Reply:
Oh my bad I was a bit tired, I will keep the abstraction only for `ActiveStorage::Representations::ProxyControllerWithVariantsTest` and will clean the entire file test in a followup PR as discussed